src/components/carousel/Carousel.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+
+import { CarouselData } from "../../data/carousel.data";
+
+const Carousel = () => {
+  const [current, setCurrent] = useState(0);
+
+  const length = CarouselData.length;
+
+  const nextSlide = () => {
+    setCurrent(current === length - 1 ? 0 : current + 1);
+  };
+
+  const prevSlide = () => {
+    setCurrent(current === 0 ? length - 1 : current - 1);
+  };
+
+  return (
+    <>
+      <div className="carousel-container">
+        {CarouselData.map(({ id, src, alt }, index) => (
+          <div
+            className={
+              index === current ? "container display-block" : "container"
```

Comment:
Here, as class `container` is common in both conditions. So you can do something like this,
```css
className={`container ${index === current ? "display-block" : ""}`}
```